include/mamba/api/construct.hpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2019, QuantStack and Mamba Contributors
+//
+// Distributed under the terms of the BSD 3-Clause License.
+//
+// The full license is in the file LICENSE, distributed with this software.
+
+#ifndef MAMBA_API_CONSTRUCT_HPP
```

Comment:
this part is really only used in conda constructor context and there we use micromamba. I think that could be moved back to micromamba?

Reply:
sure we can move it back to `micromamba` only then